### PR TITLE
.travis.yml: add pair of linux-ppc64le targets.

### PR DESCRIPTION
**Pull request by dot-asm**
WIP, no approval requested. Second commit that removes most targets will be removed prior approval is requested. The request will be closed and reopened couple of times in order to get some timing statistics.

### CODE REVIEW

**Comment by dot-asm** [comment]
It's about 2 minutes provision time. There is no support ccache-ing. There is ccache, but cache is not offloaded, so there is no gain. As result non-extended test build is couple of minutes slower than it could have been. One can also observe several minutes variation, as if jobs are schedule to less and more busy systems, unbalanced that is. Extended test can take >30 minutes (if you happen to run on apparently more busy system). This is >2x slower that tests on x86_64. One can wonder if this can have some kind of serializing effect on queue, because if it does, then push-to-repo queue could get unbearably slow during bursts...

**Comment by dot-asm** [comment]
Extended test adds between 30 to 50[!] minutes of additional CPU time. I'd say this is a bit excessive. I therefore would rather suggest to keep one non-extended --strict-warning build and keep sanitizer build commented for occasions when there is reason to believe that there is PPC-specific problem that can be troubleshooted with sanitizer... Removing WIP...

File: .travis.yml [hunk 1]
```diff
@@ -109,6 +113,12 @@ matrix:
                       - gcc-mingw-w64
           compiler: x86_64-w64-mingw32-gcc
           env: EXTENDED_TEST="yes" CONFIG_OPTS="no-pic"
+        # Uncomment if there is reason to believe that PPC-specific problem
+        # can be diagnosed with with possibly >30 mins sanitizer build...
```
Comment (FdaSilvaYY) [inline .travis.yml]: " can be diagnosed with with possibly "

Comment (dot-asm) [reply .travis.yml]: Addressed. Thanks!

**Comment by dot-asm** [comment]
Re-based and ready to roll.

**Pull request closed without merging**